osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -729,21 +733,21 @@ private void updateYPositions()
         {
             visibleItems.Clear();
 
-            float currentY = visibleHalfHeight;
+            float currentY = visibleHalfHeight - BleedTop;
 
             scrollTarget = null;
 
             foreach (CarouselItem item in root.Children)
             {
-                if (item.Filtered.Value)
-                    continue;
-
                 switch (item)
                 {
                     case CarouselBeatmapSet set:
                     {
-                        visibleItems.Add(set);
                         set.CarouselYPosition = currentY;
+                        if (item.Filtered.Value)
```

Comment:
this is SUPER weird to be here, with no comment and only in this part of the `switch`

Reply:
This part of the switch is the only part. In fact I think carousel root only hold beatmap sets.

But this is here so that we set the target Y / ``CarouselYPosition`` for even the filtered sets. They would appear at or fade into the beatmap set above them